packages/openapi-generator/src/parser/schema.ts

```diff
@@ -169,9 +169,17 @@ function parseEnumSchema(
   const type = schema.type ? getType(schema.type) : 'string';
   return {
     type,
-    enum: (schema.enum || []).map(entry =>
-      type === 'string' ? getEnumStringValue(entry) : entry
-    )
+    enum: (schema.enum || []).map(entry => {
+      if (type === 'string' && entry !== null) {
+        return getEnumStringValue(String(entry));
+      }
+      if (entry === null && !schema.nullable) {
+        throw new Error(
+          "'Null' was used as a parameter, but nullable wasn't declared"
```

Comment:
Either way I would suggest the following improvements to the message:
* Use null instead of 'Null', even in the beginning of a sentence (lower case + no quotes).
* Use "was not" instead of "wasn't".
* Add a fullstop at the end of the sentence.
* Add the notion of an enum somehow.
```suggestion
          "null was used as a parameter in an enum, although the schema was not declared as nullable"
```

Reply:
Agreed! I wasn't satisfied with the error message as well but wasn't sure what the best one would be, your approach looks clean.